src/main/java/com/hartwig/platinum/config/HmfConfiguration.java

```diff
@@ -0,0 +1,8 @@
+package com.hartwig.platinum.config;
+
+public interface HmfConfiguration {
```

Comment:
Introduced this new configuration to make it a little more explicit that this is "hmf specific" and remove the need for checking multiple optionals